modules/odc-monitoring-your-app-vulnerabilities.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="odc-monitoring-your-application-image-vulnerabilities-metrics_{context}"]
+= Monitoring your application and image vulnerabilities metrics
+
+After you create applications in your project and deploy them, use the *Developer* perspective in the web console to see the metrics for your application dependency vulnerabilities across your cluster. The metrics help you to analyze the following image vulnerabilities in detail:
+
+* Total count of vulnerable images in a selected project
+* Severity-based counts of all vulnerable images in a selected project
+* Drilldown into severity to obtain the details, such as count of vulnerabilities, count of fixable vulnerabilities, and number of affected pods for each vulnerable image
+
+.Prerequisites
+* You have installed the Container Security operator from the Operator Hub.
+
+.Procedure
+
+. For a general overview of the image vulnerabilities, on the navigation panel of the *Developer* perspective, click *Project* to see the project dashboard.
+
+. Click *Image Vulnerabilities* in the *Status* section. A pop-up opens containing the image vulnerabilities breakdown details, such as *Vulnerable Container Images* and *Fixable Container Images*.
++
+[NOTE]
+====
+The different color icons implies the severity level of vulnerabilities. Red for Critical, orange for Medium, and yellow for Low.
+
+.Viewing image vulnerabilities
+image::odc_image_vulnerabilities.png[]
+====
+
+. For detailed vulnerabilities overview, click the *Vulnerabilities* tab on the project dashboard.
+
+.. Select an *Image Name* for further drilldown to vulnerabilities details associated with that particular image.
++
+[NOTE]
+====
+You can see the total number of vulnerabilities in the *Total* column of all the listed images.
+====
+
+.. Apply filter on the vulnerabilities based on their *Severity*, *Package*, *Type*, *Source*, *Current Version*, and *Fixed in Version*.
```

Comment:
We have filters based on `Type` and `Severity`.

Users can sort the Vulnerabilities table based on `Severity`, `Package`, `Type`, `Source`, `Current Version`, and `Fixed in Version`.

<img width="417" alt="image" src="https://user-images.githubusercontent.com/2561818/180730497-d953b0a0-91a5-41e5-8774-2fadb2abd59f.png">